src/lib/discord/bot.ts

```diff
@@ -13,13 +16,119 @@ new Command({
 		required: true,
 	} as const)
 	.handle(async i => {
+		const name: string = i.options.name
+		const entry: Entry | null = await fetchEntry(createSlug(name))
+		if (!entry) {
+			return {
+				content: `Requested entry "${name}" does not exist, or is unavailable`,
+				color: 16711680,
+			}
+		}
+
+		let content: string
+		if (entry.content.length > 985) {
+			content = `${entry.content.slice(0, 985)}...`
+		} else {
+			content = entry.content
+		}
 		return {
 			embeds: [
 				{
-					title: `Entry "${i.options.name}"`,
+					title: `${name}`,
+					url: `${BASE_URL}/entry/${entry.slug}`,
+					description: content,
+					color: 16711680,
 				},
 			],
 		}
 	})
 
+new Command({
+	name: 'random',
+	description: 'View a random Repldex entry',
+}).handle(async data => {
+	console.log(data)
+	const entries = await fetchEntries({
+		limit: 1,
+		skip: Math.floor(Math.random() * (await countEntries())),
+	})
+	const entry = entries[Math.floor(Math.random() * entries.length)]
+	return {
+		embeds: [
+			{
+				title: entry.title,
+				url: process.env.BASE_URL + '/entry/' + entry.slug,
+				description: entry.content,
+				color: 16711680,
+			},
+		],
+	}
+})
+
+new Command({
+	name: 'source',
+	description: 'Get a link to the source code of Repldex',
+}).handle(data => {
+	console.log(data)
```

Comment:
This console.log probably shouldn't be here either

Reply:
This is outdated, resolving conversation